gstreamer/src/event.rs

```diff
@@ -440,6 +440,7 @@ impl glib::types::StaticType for EventRef {
 impl fmt::Debug for EventRef {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         f.debug_struct("Event")
+            .field("ptr", unsafe { &self.as_ptr() } )
```

Comment:
You're missing caps also

Reply:
It uses `self.to_string()` which should do the right thing.

Reply:
I mean the caps `Debug` impl does not print the pointer :) We should add this consistently for all miniobjects

Reply:
This is how it is implemented:

``` rs
impl fmt::Debug for CapsRef {
    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
        f.debug_tuple("Caps").field(&self.to_string()).finish()
    }
}
```

You want to make the imple use debug_struct you mean?

Reply:
Oh no, I was confused. Sorry :) The way how it was before is nicer IMHO. Will have to think about all this for a bit